db/migrate/20210630142005_add_current_version_id_to_unrecognized_appellants.rb

```diff
@@ -0,0 +1,6 @@
+class AddCurrentVersionIdToUnrecognizedAppellants < Caseflow::Migration
+  def change
+    add_reference :unrecognized_appellants, :current_version, index: false, foreign_key: { to_table: :unrecognized_appellants }, comment: "The current version ID for this unrecognized appellant"
+    add_reference :unrecognized_appellants, :created_by, index: false, foreign_key: { to_table: :users}, comment: "The ID of the user that created this version of the unrecognized appellant"
```

Comment:
"current version ID" may be misleading -- it's simply referring to the record `id`. I think it's obvious this column is storing a record id (that's the idea of a foreign key), so we don't need to explicitly state it:
```suggestion
    add_reference :unrecognized_appellants, :current_version, index: false, foreign_key: { to_table: :unrecognized_appellants }, comment: "The current version for this unrecognized appellant"
    add_reference :unrecognized_appellants, :created_by, index: false, foreign_key: { to_table: :users}, comment: "The user that created this version of the unrecognized appellant"
```